.travis.yml

```diff
@@ -9,14 +9,22 @@ branches:
 
 go: 1.11.x
 
+env:
+  global:
+    - GO_METALINTER_VERSION="v3.0.0"
+
 install:
-  - curl -L https://git.io/vp6lP | sh
+  # install gometalinter
+  - >
+    curl -L
+    'https://raw.githubusercontent.com/alecthomas/gometalinter/master/scripts/install.sh'
```

Comment:
let's pick up a released version instead of master code
https://github.com/alecthomas/gometalinter/blob/v3.0.0/scripts/install.sh

Reply:
done PTAL